distributed-job-manager/src/test/java/ru/fix/distributed/job/manager/WorkPooledMultiJobIT.java

```diff
@@ -519,6 +519,43 @@ public void shouldBalanceOnWorkPoolMultipleUpdate() throws Exception {
         }
     }
 
+    @Test
+    public void cleaning_WHEN_last_djm_with_available_job_closed_THEN_removing_job_from_workPool() throws Exception {
+        DistributedJob job1 = new StubbedMultiJob(1, getWorkItems(4));
+        DistributedJob job2 = new StubbedMultiJob(2, getWorkItems(4));
+        DistributedJob job3 = new StubbedMultiJob(3, getWorkItems(4));
+        long awaitCleaningTimeout = getWorkPoolCleanPeriod().get() * 3;
+
+        CuratorFramework curator1 = zkTestingServer.createClient();
+        DistributedJobManager jobManager1 = createNewJobManager(
+                "djm-1",
+                curator1,
+                List.of(job1, job2)
+        );
+        try (
```

Comment:
using try with resource in for djm2 and not using for jdm1 is confusing. Better not to use try-with-resource block at all.

Reply:
fixed